### DISCUSSION TITLE
Export specific clips in 0.14?

### DISCUSSION THREAD

**Discussion by thinkloop** [blakeblackshear/frigate#13483] [General]
Amazing work on 0.14 👏 This is definitely the right direction! I noticed favourites/starred videos are gone in favour of exporting. That seems alight. Am I missing something tho in that it is not possible to export a specific clip? On the review page, if you select a clip (or multiple) the only available option is to mark as reviewed - no export. When you click in to a clip, then click "Export" you are presented with a generic export dialog unrelated to the clip. 

Is there a way to export selected clips?

**Comment by thinkloop** [comment]
It seems the "Select from Timeline" does just that. It exports the selected clip. A suggestion, when a clip is selected, it would be nice for "select from Timeline" to be automatically selected, and for a default name for the file to be auto-populated, so that exports can be done quickly and thoughtlessly (like with stars previously).

**Comment by hawkeye217** [comment]
See the [FAQ](https://github.com/blakeblackshear/frigate/discussions/12724), Usage question 6.

**Reply by thinkloop** [reply]
Thank you, that's important, I never saw it because I always selected 2 or more videos for some reason, and the "export" button no longer shows up in that case. Suggestion to always show the "export" button but disable it in various cases.
